Gopkg.toml

```diff
@@ -28,7 +28,8 @@
 
 [[constraint]]
   name = "github.com/fusor/openshift-velero-plugin"
-  branch = "master"
+  branch = "route-host"
```

Comment:
Is this intentional, to not rely on 'master' but to rely on a specific branch of openshift-velero-plugin?

Reply:
@jwmatthews It's temporary. See my comment above. This is required for this PR to be tested, since it's the only way to pull in the required openshift-velero-plugin changes until the velero plugin PR is merged. My usual workflow with PRs for the migration plugin that have companion velero plugin PRs is this:
1) Push velero-plugin PR branch and create velero plugin PR
2) Point migration-plugin to the PR branch so I can build images and test locally
3) Create WIP migration-plugin PR pointing to velero-plugin PR
4) Review/merge velero-plugin PR
5) Update migration-plugin to point back to velero-plugin master
6) Review/merge migration-plugin PR.